openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -273,3 +273,10 @@
   flex-direction: column;
   padding: 24px 16px 0px 16px;
 }
+
+.description-text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  line-height: 18px;
+}
```

Comment:
@aniketkatkar97 , Why did we remove these properties?

```css
   max-height: 64px;
  -webkit-line-clamp: 3;
  -webkit-box-orient: vertical;
```

Reply:
This was older styling given to limit 3 lines for description. As we are handling that logically in the component. we don't need that styling. Also, this styling is what was causing the original issue in 'Safari' in the first place

Reply:
Okay, Can we make sure to show 3 lines in the explore page card as we had before?

Reply:
Sure, looking into that